src/components/IncidentTable/subcomponents/PersonInitialsComponents.js

```diff
@@ -46,7 +46,7 @@ const PersonInitialsComponent = ({
             overlay={<Tooltip id={`tooltip-${user.id}`}>{user.summary}</Tooltip>}
           >
             <div
-              idx={user.id}
+              data-user-id={user.id}
```

Comment:
@giranm - as far as I can see we don't actually use these `idx` values. I've swapped them to `data-*` properties to please `eslint`. Thoughts?

Reply:
For some reason I assumed I was assigning the ID to the `keys` param ([see docs](https://reactjs.org/docs/lists-and-keys.html#keys)).
`data-*` is a perfectly suitable prop to use here.